lib/features/preview-support/PreviewSupport.js

```diff
@@ -135,15 +143,18 @@ PreviewSupport.prototype.addFrame = function(shape, group) {
 
   svgAppend(group, frame);
 
+  svgAttr(frame, 'data-complex-preview-element-id', shape.id);
+
   return frame;
 };
 
 /**
  * Clone all markers referenced by a node and its child nodes.
  *
  * @param {SVGElement} gfx
+ * @param {string} [className="djs-dragger"]
  */
-PreviewSupport.prototype._cloneMarkers = function(gfx) {
+PreviewSupport.prototype._cloneMarkers = function(gfx, className = 'djs-dragger') {

```

Comment:
We're already ES6, so this should be fine, right? We could apply it in other places, i.e. https://github.com/bpmn-io/diagram-js/pull/807/files#diff-f269b8125f70b7da3aecb63f2c32665d0124edde0c515fd7bc408bf6babba8abR208, too.

Reply:
👍🏻